### ISSUE TITLE
SRW - AttributeError: 'dict' object has no attribute 'models'

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#611]
I'm trying to import this [JSON file](https://github.com/radiasoft/sirepo/files/700580/SIX.json.txt), I get the following error:
```py
..........................................
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/SocketServer.py", line 655, in __init__
    self.handle()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/werkzeug/serving.py", line 216, in handle
    rv = BaseHTTPRequestHandler.handle(self)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/BaseHTTPServer.py", line 340, in handle
    self.handle_one_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/werkzeug/serving.py", line 251, in handle_one_request
    return self.run_wsgi()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in execute
    application_iter = app(environ, start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/beaker/middleware.py", line 155, in __call__
    return self.wrap_app(environ, session_start_response)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/server.py", line 643, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/server.py", line 263, in app_import_file
    return _save_new_and_reply(simulation_type, data)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/server.py", line 733, in _save_new_and_reply
    data = simulation_db.save_new_simulation(*args)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/simulation_db.py", line 642, in save_new_simulation
    return save_simulation_json(simulation_type, data)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/simulation_db.py", line 660, in save_simulation_json
    data = fixup_old_data(data)[0]
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/simulation_db.py", line 200, in fixup_old_data
    sirepo.template.import_module(data['simulationType']).fixup_old_data(data)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/template/srw.py", line 363, in fixup_old_data
    data.models.tabulatedUndulator.indexFileName = data.models.tabulatedUndulator.indexFile
AttributeError: 'dict' object has no attribute 'models'
```

**Comment by mrakitin** [comment]
For regular run (not import):
```py
>>> type(data)
<class 'pykern.pkcollections.Dict'>
```
for importing of JSON file:
```py
>>> type(data)
<type 'dict'>
```

**Comment by robnagler** [comment]
I'll look tomorrow. I started writing a test first to reproduce.

**Comment by robnagler** [comment]
fix 01276a522c85809595a1d6da124ac53ae3eabb2a
